file1.txt

```diff
@@ -0,0 +1,2 @@
+this is test file
+adding second line
```

Comment:
Why do we need this file?

Reply:
I added some test files while doing my labs from the server.